backend/zkppSalt.js

```diff
@@ -1,27 +1,57 @@
-import { blake32Hash } from '~/shared/functions.js'
-import { timingSafeEqual } from 'crypto'
+import { timingSafeEqual, randomBytes } from 'crypto'
 import nacl from 'tweetnacl'
 import sbp from '@sbp/sbp'
 import { boxKeyPair, encryptContractSalt, hashStringArray, hashRawStringArray, hash, parseRegisterSalt, randomNonce, computeCAndHc, base64urlToBase64, base64ToBase64url } from '~/shared/zkpp.js'
+import { readFileSync } from 'fs'
+
+// Input keying material used to derive various secret keys used in this protocol
+let IKM: string
```

Comment:
Could you add more comments here to comment on whether or not this secret is relevant to data migration from one server to another?